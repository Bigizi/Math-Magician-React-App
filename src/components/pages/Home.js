import React from 'react'

function Home() {
const styles = {
    main: {
        display: "flex",
        flexDirection: "column"
    },

    h1: {
        fontSize: "28px",
        fontWeight: "bold"
    },
};
  return (
    <div className='main' style={styles.main}>
      <h1 className='h1' style={styles.h1}>Math Magicians</h1>
      <h2>Welcome to our page!</h2>
      <p>A calculator is a device that performs arithmetic operations on numbers. Basic calculators can do only addition, subtraction, multiplication and division mathematical calculations.</p>
      <p>A calculator is a small electronic device that makes mathematical calculations. It’s used by students, financial advisors, accountants, and many others when mental math won’t produce a fast, accurate answer</p>
    </div>
  )
}

export default Home
