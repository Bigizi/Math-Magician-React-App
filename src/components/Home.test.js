import renderer from 'react-test-renderer';
import Home from './Home';

describe('Test Home component', () => {
  it('Home component should match snapshot', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
