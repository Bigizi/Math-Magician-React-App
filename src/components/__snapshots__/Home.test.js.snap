// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home component Home component should match snapshot 1`] = `
<div
  className="main"
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <h1
    className="h1"
    style={
      Object {
        "fontSize": "28px",
        "fontWeight": "bold",
      }
    }
  >
    Math Magicians
  </h1>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    A calculator is a device that performs arithmetic operations on numbers. Basic calculators can do only addition, subtraction, multiplication and division mathematical calculations.
  </p>
  <p>
    A calculator is a small electronic device that makes mathematical calculations. It’s used by students, financial advisors, accountants, and many others when mental math won’t produce a fast, accurate answer
  </p>
</div>
`;
